doc/sphinx/compiling.rst

```diff
@@ -32,69 +33,267 @@ Installation Prerequisites
 Linux
 -----
 
-* For Ubuntu or Debian users, the following packages should be installed using
+General Notes
+^^^^^^^^^^^^^
+
+* To download the source code, installing ``git`` is highly recommended.
+
+* SCons is only available for Python 2, so building the Python 3 module requires
+  two installations of Python (one of Python 2 and one of Python 3), even if you
+  do not intend to build the Python 2 module.
+
+* The following instructions use the system-installed versions of Python, but
+  alternate installations such as the Anaconda distribution of Python can be
+  used as well.
+
+* Cython is only required to be installed for the version of Python that also
+  has SCons installed; following the instructions below will install Cython for
+  the version of Python 2 installed in the system directories.
+
+* The SUNDIALS suite of solvers used in Cantera is available from several sources.
+  The sources in most distribution repositories may be out of date, or not
+  optimized for your hardware. The source code for SUNDIALS will be downloaded
+  and compiled with the rest of Cantera, so installing from your distribution's
+  repositories is optional.
```

Comment:
I think the truly minimal installation (no Python module) is not very interesting, and isn't really worth spending time on. I think making this be the recommended configuration would be better, although that barely helps in distinguishing which method of SUNDIALS installation to prefer.

Probably the bigger source of confusion here (at least for me) is the discrepancy between suggesting here that users go with the bundled SUNDIALS, while the SConstruct file prefers a system installation if it can find one (and I don't want to change that this late in the release cycle for 2.3.0). I'm fine with either choice here, and we can document the caveats elsewhere.